concepts/framework/http_cache.md

```diff
@@ -52,20 +52,20 @@ Shopware uses several client cookies to differentiate the requests. This allows
 
 This cookie will be set when the non-logged-in customer with an empty cart changes the current currency. Why does Shopware need a separate cookie for currency? It allows us to maximize the cache hits for non-logged-in customers as we separate the cache as less as possible.
 
-### sw-cache-hash
+#### sw-cache-hash
 
 This cookie replaces the `sw-currency` cookie and contains the active rules and active currency. This cookie will be set when the active rules do not match the default anymore \(e.g., customer login/items in cart\).
 
-### sw-states
+#### sw-states
```

Comment:
```suggestion
### sw-states
```

Reply:
As per documentation guidelines and linter formatter, heading levels can't be skipped.